schemas/components/schemas/VpnResult.yaml

```diff
@@ -46,6 +46,12 @@ properties:
       - publicVPN
       - auxiliaryMobile
       - osMismatch
+  confidence:
+    type: string
+    description: >-
+      provides the confidence of the detected vpn based on reliability of source and algorithm
+      we will be categorising the response into low/medium/high (which are the 3 supported values for the response type)
+    example: low
 required:
```

Comment:
Is `confidence` returned for both positive and negative results? Then is should be in the listed in `required`, right? 
If not, the description should specify that it is returned only for positive results.

Reply:
Yes, will always be present 
I have added this to the required column